src/codeMirror/style.scss

```diff
@@ -0,0 +1,53 @@
+$gutterWidth: 43px; // enough for 3 chars
```

Comment:
~Not sure if this a Tony thang or not, but we should probably be using `em` values if we're trying to approximate font widths.~ Never mind — I see we're doing further SCSS-based calculations on this value, so all the math would have to be converted to CSS `calc()`, and at that point it's probably outside the scope of this PR!